build.savant

```diff
@@ -18,7 +18,7 @@ jacksonVersion = "2.13.4"
 jacksonDatabindVersion = "2.13.4+2"
 testngVersion = "7.3.0"
 
-project(group: "io.fusionauth", name: "fusionauth-scim", version: "2.2.0", licenses: ["ApacheV2_0"]) {
+project(group: "io.fusionauth", name: "fusionauth-scim", version: "2.2.1", licenses: ["ApacheV2_0"]) {
```

Comment:
While in here, let's update the savant plugins to `2.0.0-RC.6` and the `idea` one can be `2.0.0-RC.7`.

Reply:
We can upgrade jackson to `>= 2.15.3`